app/uk/gov/hmrc/apiplatformdeskpro/domain/models/DeskproPerson.scala

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.apiplatformdeskpro.domain.models
+
+import play.api.libs.json.{Json, OFormat}
+
+case class DeskproPerson(
+    name: String,
+    email: String
+  )
+
+object DeskproPerson {
+  implicit val ticketFormat: OFormat[DeskproPerson] = Json.format[DeskproPerson]
```

Comment:
Should this be `personFormat` or just `format`?
Also, the formatter is duplicated in the `CreateTicketRequest`, line 33.